doczrc.js

```diff
@@ -47,30 +47,29 @@ export default {
   },
   menu: [
     "Introduction",
+    "Roadmap",
     {
-      name: "Identity",
+      name: "DSNP",
       menu: [
```

Comment:
Some of this should really be nested deeper, but docz does not support sub-nested menus #141. For now, I think this is fine until that issue gets resolved.